src/serve.c

```diff
@@ -585,6 +585,8 @@ static int dcc_run_job(int in_fd,
     char *server_cwd = NULL;
     char *client_cwd = NULL;
     int changed_directory = 0;
+    char * out_fname_gcno=NULL,*tmpcopy=NULL;
+    int hasgcov=0;
```

Comment:
const?

Reply:
It must be set again, so can not be const.